src/run_statistics.rs

```diff
@@ -4,11 +4,23 @@ use std::{fmt, time::Duration};
 pub struct RunStatistics {
     pub runtime: Duration,
     pub threshold: Duration,
+    pub logs: String,
+    pub output: serde_json::Value,
```

Comment:
Conceptually these aren't statistics to me- I think we're missing a parent `Result` concept that would own the logs, output, and statistics.

Reply:
I think that decision comes down to whether or not there's value in separating the display portion into separate modules for logs/output and statistics or if it's easier to just have one display implementation for the entire result- I don't have a strong opinion on this 🤷🏻